bin/cmd.js

```diff
@@ -38,13 +38,21 @@ function bundle () {
     var wb = w.bundle();
     
     w.pipeline.get('pack').once('readable', function() {
-        wb.pipe(writer);
+        wb.pipe(writer).on('error', function (err) {
```

Comment:
The [return value](https://github.com/nodejs/node/blob/11ad744a92374ad71730cbfb7abea71fda0abb74/lib/stream.js#L106) of `.pipe` is the destination stream. So this is equivalent to:

```js
wb.pipe(writer);
writer.on('error', /*...*/);
```

If the `writer` has an error, its best to let watchify die (when streams emit an `error` event and there is no `error` handler, an uncaught exception happens and the app crashes).

If you can remove this, and squash your commits, I'll happily merge it.